tasks/update-code/git.yml

```diff
@@ -1,6 +1,30 @@
 ---
+
+- name: ANSISTRANO | GIT | Ensure GIT deployment key is up to date
+  copy:
+    src: "{{ ansistrano_git_identity_key_path }}"
+    dest: ".ssh/git_identity_key"
```

Comment:
yes, `dest` will be relative to the remote_user $HOME directory

I see your point but since the key is overwritten every time it will be fine (although the file itself may change in the scenario you are describing) although it clearly violates the idea of things being idempotent :)

Can you think of any implications of changing this to use a full path to store the ssh key using the `{{ ansistrano_deploy_to }}` variable?
Or maybe setting some destination filename so that the key is still stored in .ssh but not conflicting with other ansible playbooks deploying to the same server? Do you know if there is any internal ansible variable that might be used for something like this?